agent/roles/manifests/templates/install-config_yaml.j2

```diff
@@ -12,8 +12,9 @@ controlPlane:
   replicas: {{ num_masters }} 
 metadata:
   name: {{ cluster_name }} 
+{% if boot_mode != "DISKIMAGE" %}
   namespace: {{ cluster_namespace }}
-networking:
+{% endif %}networking:
```

Comment:
The formatting seems a bit odd, shouldn't `networking` be on a separate line?

Reply:
oh sorry, good catch